mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -23,6 +23,7 @@ CLASS net/minecraft/class_1263 net/minecraft/inventory/Inventory
 	METHOD method_5443 canPlayerUseInv (Lnet/minecraft/class_1657;)Z
 		ARG 1 player
 	METHOD method_5444 getInvMaxCountPerStack ()I
+		COMMENT @return the max {@link net.minecraft.item.ItemStack#count count} of item stacks in this inventory
```

Comment:
You don't need to repeat `count` in the link

Reply:
`return the max ItemStack#count of item stacks in this inventory` ?

Reply:
since the count field is private, you should refer to the getter method.

Reply:
If you refer to `getCount`, you need to repeat "count" in the link for it to look nice (maybe also use `@linkplain`). 🤔